danieldupriest/reimbursinator
# Update section api

Each section is successfully being updated after entering data in the front-end - every field except file upload should be working

Testing
1. Run docker and log in with `user@user.com` password `user12345`
2. Edit report --> `NEW REPORT` --> fill out random fields in the report 
*NOTE*: The fields requiring a date are auto-populated with None's - delete these and leave it blank. Django doesn't like sending null's back to the db and will throw an error. The front end should be able to hide this? 
3. Click save section, a dialogue box should pop up indicating success.
4. Exit out of the report and click it again - and your saved data will be visible.
5. Try testing out by creating a new report too
